docs/sources/enterprise/reporting.md

```diff
@@ -145,12 +153,24 @@ The last saved version of the report will be sent to selected emails. You can us
 
 {{< figure src="/static/img/docs/enterprise/reports_send_test_mail.png" max-width="500px" class="docs-image--no-shadow" >}}
 
-## Pause report
+### Pause report
 
 > **Note:** Available in [Grafana Enterprise]({{< relref "../enterprise" >}}) version 8.0 and later, and [Grafana Cloud Pro and Advanced]({{< ref "/docs/grafana-cloud" >}}).
 
 You can pause sending of reports from the report list view by clicking the pause icon. The report will not be sent according to its schedule until it is resumed by clicking the resume button on the report row.
 
+### Add multiple dashboards to report
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../enterprise" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
+
+You can add more than one dashboard to the report, which will be rendered in the same PDF file or image. It is currently not possible to add the same dashboard more than once to the report.
+
+### Embed dashboard as an image into report
+
+> **Note:** Available in [Grafana Enterprise]({{< relref "../enterprise" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).
+
+You can send a report email with an image of the dashboard embedded in the email itself, instead of attached as a PDF. In this case the email recipients can see the dashboard at a glance instead of having to open the PDF.
+
```

Comment:
```suggestion
You can add more than one dashboard to the report, which will be rendered in the same PDF file or image. It is currently not possible to add the same dashboard more than once to the report.

### Embed dashboard as an image into report

> **Note:** Available in [Grafana Enterprise]({{< relref "../enterprise" >}}) version 9.0 and later, and [Grafana Cloud Pro and Advanced]({{< relref "/grafana-cloud" >}}).

You can send a report email with an image of the dashboard embedded in the email itself, instead of attached as a PDF. In this case, the email recipients can see the dashboard at a glance instead of having to open the PDF.

```